sdk/python/foundation-models/system/finetune/chat-completion/chat-completion.ipynb

```diff
@@ -0,0 +1,731 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Chat Completion - Ultrachat-200k\n",
+    "\n",
+    "This sample shows how use `chat-completion` components from the `azureml` system registry to fine tune a model to complete a conversation between 2 people using ultrachat_200k dataset. We then deploy the fine tuned model to an online endpoint for real time inference.\n",
+    "\n",
+    "### Training data\n",
+    "We will use the [ultrachat_200k](https://huggingface.co/datasets/HuggingFaceH4/ultrachat_200k) dataset. This is a heavily filtered version of the UltraChat dataset and was used to train Zephyr-7B-β, a state of the art 7b chat model.\n",
```

Comment:
We can talk about the training data during the download and visualization section. Just keep the summary here and move the details to the above-mentioned section.